dts/bindings/interrupt-controller/st,stm32g0-exti.yaml

```diff
@@ -5,7 +5,7 @@
 description: |
         STM32 controller
         This compatible stands for all interrupt-controller blocks for the
-        STM32C0/G0/L5/mp/u5.
+        STM32C0/G0/H5/L5/MP1/U5.
```

Comment:
Let's use some generic wording so we don't have to update this for each new series added.

Reply:
This is the description introduced by the https://github.com/zephyrproject-rtos/zephyr/pull/55844 that is updated there.